packages/web-twig/src/Resources/components/FileUploader/stories/FileUploaderAttachmentWithImagePreview.twig

```diff
@@ -0,0 +1,48 @@
+<section class="docs-Section">
+    <h2 class="docs-Heading">Attachment with Image Preview</h2>
+
+    <Modal id="example_image_preview" titleId="example_image_preview_title">
+        <ModalDialog>
+            <ModalHeader modalId="example_image_preview" titleId="example_image_preview_title">
+                Image Preview
+            </ModalHeader>
+            <ModalBody>
+
```

Comment:
👀

Reply:
Please check the React implementation, I think I suggested some padding around the image…

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/7fe0246f-a9ac-4fa3-8e51-eb67087bc9af)